### DISCUSSION TITLE
Proof of enrollment is not recognized by github education pack

### DISCUSSION THREAD

**Discussion by Douglas-Wallace** [community/community#144693] [GitHub Education]
### Hi everyone,

I'm trying to purchase the Education Pack, but I've already submitted my institution's registration document twice, and both times it was rejected for no apparent reason. The institution is recognized by the website itself, as it even suggests the institution when I enter my email

**Comment by pikaping** [comment]
Hello!

Which kind of proof are you trying to submit? Currently they are accepting Student ID, Transcript, Enrollment, Affiliation letter or Class schedule. Also have you checked that your document has at least your first and last name, complete school name, and a date demonstrating current academic affiliation must appear in your document?

Thanks!

**Reply by Douglas-Wallace** [reply]
I'm using a proof of enrollment document that includes my full name, student ID, issuance details (dated yesterday), and the course I'm enrolled in. The only unusual aspect I can think of is that my institutional email refers to the parent organization rather than the college itself. This is because the college is governed by an institution that has its own domain, so students are provided with emails from the institution rather than directly from the college

**Reply by pikaping** [reply]
have you verified your college email? i had this same problem with my last institution (we had @student.university.edu) but it didn't interfiere. I suggest trying with another document if possible and check the submission quality.

**Reply by Douglas-Wallace** [reply]
I’ve already tried using the document from the institution, which is what I have, but now I’ll try submitting one from the college along with my academic curriculum. I took a screenshot from the college’s institutional website that includes all the required information. Additionally, I highlighted all the relevant data with a blue circle to make it easier for them to locate

**Comment by davevad93** [comment]
Hi @Douglas-Wallace ,

I once had the same problem, at first I submitted my proof of enrollment and it was rejected, then I submitted a screenshot of my last exam results showing the name of the school, my full name and a current date in a `DD/MM/YYYY` format and it was finally accepted.

As stated in the [docs](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-education-for-students/apply-to-github-education-as-a-student), ***Documents that prove your current student status include a picture of your school ID with current enrollment date, class schedule, transcript, and affiliation or enrollment verification letter.***

So try any of the above documents, just make sure that the name of the institution, your full name and a current date that proves your student status also appear on the documents to submit.

**Reply by Douglas-Wallace** [reply]
Thanks! I will try other means

**Reply by davevad93** [reply]
You're welcome @Douglas-Wallace 👍

**Comment by BancoCentraI** [comment]
> ### Oi pessoal,
> Estou tentando comprar o Education Pack, mas já enviei o documento de registro da minha instituição duas vezes, e nas duas vezes ele foi rejeitado sem motivo aparente. A instituição é reconhecida pelo próprio site, pois ele até sugere a instituição quando eu insiro meu e-mail

Parece que você está enfrentando dificuldades para obter o Education Pack, mesmo com a instituição sendo reconhecida no site. Aqui estão algumas sugestões para resolver o problema:

Verifique os Detalhes do Documento: Certifique-se de que o documento enviado está legível e contém todas as informações necessárias, como o nome completo da instituição e o logo. Além disso, verifique se ele está em um formato aceito pelo sistema.

Formato e Tamanho do Arquivo: Verifique se o arquivo enviado está dentro dos limites de tamanho e está no formato correto (geralmente PDF, PNG ou JPEG). Às vezes, documentos com alta resolução ou formatos errados podem ser rejeitados automaticamente.

Detalhes do E-mail: Como você mencionou que o sistema sugere sua instituição ao inserir o e-mail, garanta que o endereço de e-mail fornecido está associado corretamente ao domínio da instituição (por exemplo, algo como nome@universidade.edu). Isso pode ajudar a validar sua afiliação de forma mais eficiente.

Entre em Contato com o Suporte: Se os documentos estão em conformidade e ainda assim estão sendo rejeitados, o ideal seria entrar em contato com o suporte ao cliente da plataforma onde você está tentando adquirir o Education Pack. Eles podem fornecer mais detalhes sobre o motivo da rejeição e ajudar a resolver o problema.
